Makefile

```diff
@@ -189,7 +189,8 @@ check-deps:
 
 # CAAS related targets
 DOCKER_USERNAME            ?= jujusolutions
-JUJUD_STAGING_DIR          ?= /tmp/jujud-operator
+DOCKER_STAGING_DIR         ?= ${GOPATH}/tmp
```

Comment:
Will `$HOME` do it? That's where I have been creating it.

Reply:
So $HOME/tmp and $GOPATH/tmp would both work.
The reason I chose GOPATH is because we know that 'make' is already creating objects in $GOPATH (as part of $GOPATH/bin and $GOPATH/pkg). If they redirected GOPATH outside of HOME (say because of disk size, etc), I don't *know* that $HOME is being written to.